### DISCUSSION TITLE
Power Apps data source and local storage

### DISCUSSION THREAD

**Discussion by antoniojoao1** [microsoft/Power-Fx#2274] [Q&A]
I'm building an app for diary activity control in my workspace. Users need to fill out a form containing information relevant to the coordination team. I want to save these forms within Power Apps before sending them to SharePoint. This allows users to review, edit, or remove entries before final submission.

I used the "Collect" function to gather form data and store it in a local collection. This collection is then linked to a browse gallery. However, when I try to connect the browse gallery to the edit form and form viewer, I encounter two identical errors: "Invalid function. Compatible with data source value awaited." I've tried various solutions, but none have been successful. Any help would be greatly appreciated.

**Comment by anderson-joyle** [comment]
Hi. I suggest you to post questions about Power Apps at https://powerusers.microsoft.com/t5/Building-Power-Apps/bd-p/PowerAppsForum1.
